# holds data access exceptions

class DataAccessError(Exception):
    pass